remo/base/static/base/js/remolib.js

```diff
@@ -36,6 +36,14 @@ function hash_set_value(key, value) {
     }
 
     $(location).attr('hash', hash);
+
+    (function() {
+        var link = document.createElement('link');
```

Comment:
Creating a new link on each ``hash`` change leads to multiple duplicate links in ``head``. I attached a screenshot after some hash changes.
![selection_021](https://cloud.githubusercontent.com/assets/835321/7797545/31d1b51c-02f6-11e5-92c1-aacb5cc3d6a7.png)